platform/admin/core/admin-core-migration-commands/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -17,6 +17,16 @@
            xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.0.0">
     <ext:property-placeholder />
 
+    <bean id="pathBuilder" class="org.codice.ddf.platform.util.PathBuilder">
```

Comment:
I see, so we had to build own own path builder for blueprint arguments? Should work, but wish we didn't have to make a custom class for it.